fele-client-service/utils/schema-validator.js

```diff
@@ -0,0 +1,19 @@
+const ZSchema = require("z-schema");
+const validator = new ZSchema();
+
+// simple javascript function that returns a object based on validation of json schema
+const validateSchema = async (jsonSchema = {}, jsonData = {}) => {
+  try {
+    const jsonIsValid = validator.validate(jsonData, jsonSchema); // This is used to validate the schema. Returns err object or valid object
+    const errors = validator.getLastErrors();
+
+    if (jsonIsValid) return { errors: false }; // return object that JSON data is valid
+    if (errors && errors.length) return { errors: true, message: JSON.stringify(errors.map((error) => { return { "message": error.message }})) };  // return the object of errors
```

Comment:
how does the returned object looks like ?
{
errors: true,
message: [
{message: "something"},
{message: ''some thing}'
]
}

Reply:
It will look similar to the above, just that the message array would be a stringified array